packages/components/credentials/PineconeApi.credential.ts

```diff
@@ -16,11 +16,6 @@ class PineconeApi implements INodeCredential {
                 label: 'Pinecone Api Key',
                 name: 'pineconeApiKey',
                 type: 'password'
```

Comment:
I dont think we want to remove this `environment`. As some are still using it. A better way is to create a separate credential file specifically for the serverless

Reply:
@HenryHengZJ I've verified it and also read their documentation, they were keeping the compatibility, so even if you use the old index (pod index) without providing the environment it is working as expected. Their new SDK doesn't require from you to provide this param since they have new endpoint to get your index data (including environment) by api key and name